File: binder/src/models/salary.rs
use sqlx::FromRow;

use super::enums::ContractType;

#[derive(Debug, FromRow)]
pub struct Salary {
    pub salary_id: i32,
    pub contract_type: ContractType,
    pub salary_min: Option<f32>,
    pub salary_max: Option<f32>,
    pub job_ad_id: i32,
}

File: JobReadApi/JobReadApi.Application/Entities/Salary.cs
using JobReadApi.Application.Enums;

namespace JobReadApi.Application.Entities;

public class Salary
{
    public int SalaryId { get; set; }
    public ContractType ContractType { get; set; }
    public float? SalaryMin { get; set; }
    public float? SalaryMax { get; set; }
    public int JobAdId { get; set; }

    public JobAd JobAd { get; set; } = null!;
}

